src/esm_runscripts/compute.py

```diff
@@ -1,31 +1,52 @@
+import copy
+import hashlib
 import os
+import pathlib
 import shutil
 import subprocess
 import sys
-import copy
-import pathlib
 
 import f90nml
 import questionary
 import yaml
-from colorama import Fore, Back, Style, init
+from colorama import Back, Fore, Style, init
+from loguru import logger
 
-import esm_tools
 import esm_calendar
 import esm_parser
 import esm_runscripts
+import esm_tools
 
 from .batch_system import batch_system
 from .filelists import copy_files, log_used_files
 from .helpers import end_it_all, evaluate, write_to_log
 from .namelists import Namelist
-from loguru import logger
```

Comment:
Only "new" thing is `hashlib`, the rest is just `isort` doing its thing. At some point I really should show the team [`pre-commit`](https://pre-commit.com)....